assets/js/shortcut-key.js

```diff
@@ -0,0 +1,13 @@
+// Check if the user is on a Mac and update the shortcut key for search accordingly
+(function () {
+  var isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```

Comment:
```suggestion
  let isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```
I think, this could also be let instead of var.